### PR TITLE
Fix image sizes

### PR DESCRIPTION
**Pull request by ButeraMV**
LIST OF CHANGES:
- Added CSS classes to resize images
-
-
-

THINGS I'M UNSURE ON:
-
-
-

RSPEC GREEN?
YES

*MENTION GROUP MEMBER TO REVIEW PR*

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,13 @@
  *= require_tree .
  *= require_self
  */
+
+ .display {
```
Comment (snayrouz) [inline app/assets/stylesheets/application.css:17]: styling KING

File: app/views/orders/show.html.erb [hunk 2]
```diff
@@ -13,7 +13,7 @@
     </tr>
   <% @order_items.each do |order_item| %>
     <tr>
-      <td><%= image_tag(order_item.item.image) %></td>
+      <td><%= image_tag(order_item.item.image, class: 'thumbs') %></td>
```
Comment (snayrouz) [inline app/views/orders/show.html.erb:16]: NICE!

**Review by blsrofe — APPROVED** [review]
Awesome! I've had this issue before. So glad I know ho w to fix this now!

**Comment by snayrouz** [comment]
GREAT JOB MICHAEL B!

**Pull request merged**